libs/blocks/section-metadata/section-metadata.js

```diff
@@ -12,7 +12,7 @@ function handleBackground(div, section) {
   }
 }
 
-function debounce(func, timeout = 300) {
+export function debounce(func, timeout = 300) {
```

Comment:
I'm not a fan of simply exporting a function from an unrelated block so that it can be reused elsewhere in a totally different block. I'd rather see this function being moved someplace else (maybe `utils/utils.js`?).

Reply:
Moved the function to utils.js ✅